Makefile

```
@@ -5,6 +5,10 @@ build:
 test:
 	go test -v -race ./... -timeout 40m
 
+# traces-dataset.sz is used by ./pkg/tests/end_to_end_tests/ingest_trace_test.go
+download-trace-dataset:
```

```suggestion
pkg/tests/testdata/traces-dataset.sz:
```

The above would allow to use `make` internal mechanisms and allow to not use phony target.